web/renderer/.gitignore

```diff
@@ -0,0 +1,48 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I think you should just have one `.gitignore` in web. Can do like **/node_modules to capture children files/folders